bin/all_sky_search/pycbc_fit_sngls_over_multiparam

```diff
@@ -68,7 +69,6 @@ def smooth_templates(nabove, invalphan, ntotal, template_idx,
         Third float: the smoothed total count in template value
 
     """
-    if weights is None: weights = numpy.ones_like(template_idx)
```

Comment:
This change allows numpy.average to revert to numpy.mean, which saves some cost on this operation